notebooks/9_timeseries.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>At this point in the code, you have already renamed "date.utc" to "datetime".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I'll make sure the source data is using datetime in order to have this as column label across all tutorials.</p>